### PR TITLE
Modify MultiRNNCell problems

### PR DESCRIPTION
**Pull request by IamSSam**
While studying RNN lecture yesterday, I had some problems.
This is my error
error

IDE pointed below code.
`cell = rnn.MultiRNNCell([cell] * 2, state_is_tuple=True)`

I don't know exactly what problem is. So I got help from stackoverflow and I could solve this problem.
ref: http://stackoverflow.com/questions/42669578/tensorflow-1-0-valueerror-attempt-to-reuse-rnncell-with-a-different-variable-s
This problem looks like a tensorflow version problem.

### CODE REVIEW

**Comment by hunkim** [comment]
@kkweon Do you think this is necessary?

File: lab-12-4-rnn_long_char.py [hunk 1]
```diff
@@ -39,12 +40,16 @@
 X_one_hot = tf.one_hot(X, num_classes)
 print(X_one_hot)  # check out the shape
 
+
 # Make a lstm cell with hidden_size (each unit output vector size)
-cell = rnn.BasicLSTMCell(hidden_size, state_is_tuple=True)
-cell = rnn.MultiRNNCell([cell] * 2, state_is_tuple=True)
+def lstm_cell():
+    cell = tf.contrib.rnn.BasicLSTMCell(hidden_size, state_is_tuple=True)
```
Comment (kkweon) [inline lab-12-4-rnn_long_char.py]: You can see `rnn` was alreaday imported
```python
from tensorflow.contrib import rnn
```
Hence, no need to call `tf.contrib.rnn` again.
```python
def lstm_cell(hidden_size):
    # state is tuple is default to True
    return rnn.BasicLSTMCell(hidden_size)
```

File: lab-12-4-rnn_long_char.py [hunk 2]
```diff
@@ -39,12 +40,16 @@
...
+    return cell
+
+multi_cells = tf.contrib.rnn.MultiRNNCell([lstm_cell() for _ in range(2)], state_is_tuple=True)
```
Comment (kkweon) [inline lab-12-4-rnn_long_char.py]: Same here
```python
multi_cells = rnn.MultiRNNCell([lstm_cell(hidden_size) for _ in range(2)])
```

Comment (IamSSam) [inline lab-12-4-rnn_long_char.py]: Oh. I did not see import line..
Sorry for my mistakes.

**Pull request merged**